packages/ckeditor5-widget/src/widgetresize/sizeview.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module widget/widgetresize/sizeview
+ */
+
+import View from '@ckeditor/ckeditor5-ui/src/view';
+
+/**
+ * A view displaying the proposed new element size during the resizing.
+ *
+ * @extends {module:ui/view~View}
+ */
+
+export default class SizeView extends View {
+	constructor() {
+		/**
+		 * The position of the view defined based on the host size and active handle position.
+		 *
+		 * @private
```

Comment:
[This should be protected when tested.](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/code-style.html#accessibility)

Reply:
Also `@readonly`, no one will ever write to this property.

Reply:
It is not used in tests anymore (), so I made it @Private.

> Asserting viewPosition makes little sense here IMO because it is an internal component state for binding purposes only. The only thing that matters for this component is the class.